app/src/main/java/com/hedvig/app/feature/keygear/ui/ReceiptActivity.kt

```diff
@@ -199,7 +208,7 @@ class ReceiptActivity : BaseActivity(R.layout.activity_receipt) {
                 out.close()
                 file.absolutePath
             } catch (e: Exception) {
-                Timber.e(e, "Error saving image")
+                e { "$e Error saving image" }
```

Comment:
These calls are not the same. Do the following:

```suggestion
                e(it) { "Error saving image" } 
```